must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -68,6 +68,7 @@ NOOBAA_POSTGRES_LABEL='noobaa-db in (postgres)'
 for pod in $(oc -n "${ns}" get pods -l "${NOOBAA_POSTGRES_LABEL}" | grep -v NAME | awk '{print $1}'); do
     dbglog "collecting noobaa db pod logs from ${ns}"
     { timeout 120 oc -n "${ns}" logs --all-containers "${pod}" &>"${LOG_DIR}"/"${pod}".log; } >>"${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
+    { timeout 120 oc -n "${ns}" cp "${pod}":/var/lib/pgsql/data/userdata/log "${LOG_DIR}"/"${pod}"/pg_log; } >>"${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
```

Comment:
@yati1998 Do we use cp command to copy the logs? What's the best way to do this?

Reply:
We haven't used that before so not sure if it works. @alphaprinz did you try collecting the logs and see if it works? If not you can use oc rsync to copy files from the pod to the local directory.

Reply:
It worked ok in my manualy test.
I should note that these are regular files and not text printed to standard output (so their content is not captured by the "oc logs" command).